### ISSUE TITLE
java.lang.NoClassDefFoundError: CompilationUnit

### ISSUE THREAD

**Issue by Buggaboo** [xtext/xtext-gradle-plugin#67]
[I described the issue here](https://github.com/tobykurien/Xtendroid/issues/128)

com.google.common.util.concurrent.ExecutionError: java.lang.NoClassDefFoundError: Lorg/eclipse/jdt/internal/compiler/batch/CompilationUnit;

**Comment by oehme** [comment]
You only installed half the plugin, see my response on the linked issue.

**Comment by Buggaboo** [comment]
Hi, it's not fixed yet. I tried your suggestion.

**Comment by tmtron** [comment]
I also get this error with plugin version 1.0.16 when I run the gradle build:
> Execution failed for task ':app:generateDebugXtext'.
> > com.google.common.util.concurrent.ExecutionError: java.lang.NoClassDefFoundError: Lorg/eclipse/jdt/internal/compiler/batch/CompilationUnit;

as a workaround I've added jdt.core to the dependencies in my app build.gradle file:
```
dependencies {
    xtextLanguages 'org.xtext.example.mydsl:org.xtext.example.mydsl:1.0.1-SNAPSHOT'
    // https://mvnrepository.com/artifact/org.eclipse.jdt/org.eclipse.jdt.core
    xtextLanguages 'org.eclipse.jdt:org.eclipse.jdt.core:3.12.2'
)
```
shouldn't the org.xtext.android.builder plugin add this dependency automatically?

**Comment by oehme** [comment]
This is a problem in the `pom.xml` of `org.eclipse.xtext.java`. It needs JDT, but doesn't declare that dependency. This was fixed in Xtext 2.11.0.RC1.

**Comment by oehme** [comment]
I added the dependency to the builder for better backwards compatibility.
